packages/algob/src/internal/util/deployer.ts

```diff
@@ -0,0 +1,69 @@
+import { mkAccountIndex } from "../../lib/account";
+import { AlgoOperator } from "../../lib/algo-operator";
+import { loadASAFile } from "../../lib/asa";
+import { loadCheckpointsRecursive } from "../../lib/script-checkpoints";
+import type {
+  Accounts,
+  AlgobDeployer,
+  AlgobRuntimeEnv,
+  ASADefs,
+  CheckpointRepo
+} from "../../types";
+import {
+  DeployerDeployMode,
+  DeployerRunMode
+} from "../deployer";
```

Comment:
this can be put in a single line as there are only two imports

Reply:
Alright